src/main/java/org/eclipse/microprofile/starter/addon/microprofile/servers/model/SupportedServer.java

```diff
@@ -44,6 +45,7 @@
             , "-Dswarm.port.offset=100" //jarParameters
             , "8080" //portServiceA
             , "8180" //portServiceB
+            , MicroProfileVersion.MP22 // MP Spec for Java 11 support
```

Comment:
Will this become a list or it just lists the first MP version that supports Java 11 and assume all later MP specs will support Java 11?

Reply:
@Emily-Jiang IMHO it is the *first* MP supported; assumed that subsequent versions keep supporting Java 11.

Reply:
ok